airflow/providers/amazon/aws/hooks/eks.py

```diff
@@ -360,14 +360,19 @@ def get_cluster_state(self, clusterName: str) -> ClusterStates:
         :rtype: ClusterStates
         """
         eks_client = self.conn
+        result = None
 
         try:
-            return ClusterStates(eks_client.describe_cluster(name=clusterName).get('cluster').get('status'))
+            result = ClusterStates(eks_client.describe_cluster(name=clusterName).get('cluster').get('status'))
```

Comment:
This seems like a change for change's sake -- the previous version should have been okay, so lets stick with that please.

Reply:
Dont really want to change anything 😄  , these are the errors and in the except there is no return for the if clause.
                                                                 ^
airflow/providers/amazon/aws/hooks/eks.py:352: error: Missing return statement
        def get_cluster_state(self, clusterName: str) -> ClusterStates:
        ^
airflow/providers/amazon/aws/hooks/eks.py:370: error: Missing return statement
        def get_fargate_profile_state(self, clusterName: str, fargateProfileName: str) -> FargateProfileStates:
        ^
airflow/providers/amazon/aws/hooks/eks.py:396: error: Missing return statement
        def get_nodegroup_state(self, clusterName: str, nodegroupName: str) -> NodegroupStates: